### PR TITLE
l10n: Add a space in generalsettings.ui

### PR DESCRIPTION
**Pull request by Valdnet**

### CODE REVIEW

File: src/gui/generalsettings.ui [hunk 1]
```diff
@@ -222,7 +222,7 @@
         <item>
          <widget class="QPushButton" name="debugArchiveButton">
           <property name="text">
-           <string>Create Debug Archive...</string>
+           <string>Create Debug Archive ...</string>
```
Comment (er-vin) [inline src/gui/generalsettings.ui:225]: My fault there... Could you also please switch it to a proper ellipsis instead of three dots like I stupidly did?

**Review by er-vin — CHANGES_REQUESTED** [review]

File: src/gui/generalsettings.ui (hunk 1 above)
Comment (Valdnet) [reply src/gui/generalsettings.ui]: I changed 😉.

Comment (er-vin) [reply src/gui/generalsettings.ui]: Thanks!

**Review by er-vin — APPROVED** [review]

**Pull request merged**